test/test_live_reload_template.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "helper"
+
+class TestLiveReloadTemplate < JekyllUnitTest
+  context "rendering livereload script with different options" do
+    context "with port only" do
+      subject { LiveReloadTemplate.new("livereload_port" => "35729") }
```

Comment:
should this be `35_729` like its in other tests..?

Reply:
OK, I'll change!